dev/io.openliberty.microprofile.telemetry.1.0.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -58,42 +58,42 @@
         <dependency>
             <groupId>org.eclipse.microprofile.telemetry.tracing</groupId>
             <artifactId>microprofile-telemetry-tracing-tck</artifactId>
-            <version>1.0-SNAPSHOT</version>
+            <version>1.0-RC6</version>
```

Comment:
With this change you should also be able to enable the TCK running in the build.